lib/bulk_data_test_kit/export_cancel_tests.rb

```diff
@@ -13,7 +13,7 @@ module BulkDataExportCancelTests
                    :bulk_export_url
 
     def perform_export_cancel_test
-      use_token = !bearer_token.blank?
+      use_token = !smart_auth_info.access_token.blank?
```

Comment:
Is the manual addition of the access token needed anymore in these tests? I see that the fhir client is set up with the smart_auth info, regardless of if it is blank or not, which makes me think that the manual check and addition of an access token is no longer needed, since it was here before since a blank access token could not be added to the fhir_client

Reply:
I might be able to trim it in a few cases, but are you saying we should remove the entire use_token/requires_access_token feature? It [seems to come from the IG](http://hl7.org/fhir/uv/bulkdata/STU1.0.1/export/index.html#response---complete-status) but I'm not sure.

Reply:
I don't think the bulk data tests are relying on the fhir client.

Reply:
Are you sure? There is a `fhir_client` defined in the bulk data test suite, and it looks to me that the bearer token is manually being added to the export requests, which would use this fhir_client which now has the smart_auth info attached. Maybe I am misunderstanding something but it seems to me that the bulk data tests are using the fhir client and that now they are using smart_auth info, which includes the bearer token. Just wanted to double check just in case!

Reply:
Nevermind you are right, I see it actually is using the http client, which does not have auth_info attached